### PR TITLE
Override contact links in plan confirmation and plan status - [PLAT-35249]

### PR DESCRIPTION
**Pull request by nc-gibsonj**
This is dependent upon incoming design-objects change and https://github.com/BLC/nextcapital-ui/pull/5839 

image
image

### CODE REVIEW

**Comment by nc-sjoquistn** [comment]
@nc-kellya do we want punctuation at the end there? probably

File: scripts/config/copy.js [hunk 1]
```diff
@@ -190,13 +190,21 @@ _.merge(window.contentCopy.copy, {
       "pendingEnrollmentSecondaryText": "Please allow 3-4 business days for your documents to be processed. Changes cannot be made to your plan while your enrollment is pending",
     }
   },
+  "planConfirmationConfig": {
+    "bodyFooter": {
+      "text": "Have questions about your new plan? Call us at $phone"
```
Comment (nc-sjoquistn) [inline scripts/config/copy.js]: Call us at $phone`.`

**Comment by nc-sjoquistn** [comment]
@nc-fergusona noice gif lol

File: scripts/config/copy.js (hunk 1 above)
Comment (nc-kellya) [reply scripts/config/copy.js]: Yep!

**Comment by nc-gibsonj** [comment]
Added punctuation

image
image

**Pull request merged**